modules/ocf_ocfweb/manifests/dev_config.pp

```diff
@@ -11,6 +11,11 @@
   $broker = "redis://:${redis_password}@admin.ocf.berkeley.edu:6378"
   $backend = $broker
 
+
+  # installed in extrapackages but this is an dependency
+  # of the crypto libraries used in ocfweb as well
+  package { 'libcrack2-dev':; }
```

Comment:
this is already installed by ocf::extrapackages, this will conflict if you have both of these manifests in use on the same host (which we do in some cases)

Reply:
:(

I guess I'll add a note to the docs on ocfweb instead and delete the resource declaration while keeping the comment so anyone who happens to read this in the future knows that's something they need to do.

Reply:
you can just use ensure_packages instead: https://github.com/puppetlabs/puppetlabs-stdlib#ensure_packages

Reply:
oh interesting